native/src/test/java/io/ayash/blockingqueue/BlockingQueueTest.java

```diff
@@ -12,76 +12,76 @@ class BlockingQueueTest {
     @Test
     void checkNullPointerInOffer() {
         BlockingQueue queue = new BlockingQueue(5);
-        assertThrows(NullPointerException.class, () -> BlockingQueue.offer(queue, null));
+        assertThrows(NullPointerException.class, () -> queue.offer( null));
```

Comment:
```suggestion
        assertThrows(NullPointerException.class, () -> queue.offer(null));
```
check for other places as well